lib/galaxy/tools/__init__.py

```diff
@@ -441,8 +441,8 @@ def sa_session( self ):
 
     @property
     def tool_version( self ):
```

Comment:
Crazy idea: would it make sense to rename this property to `lineage` and remove:
```python
tool.lineage = tool_lineage
```
from `AbstractToolBox._load_tool_tag_set()` method in `lib/galaxy/tools/toolbox/base.py` ?

I think this property is only used and would need to be renamed:
- in the next method `tool_versions`
- in this file at line1835
- in `lib/galaxy/webapps/galaxy/controllers/admin_toolshed.py` at line 2064.